be1-go/channel/consensus/mod.go

```diff
@@ -249,6 +249,11 @@ func (c *Channel) processConsensusElect(data messagedata.ConsensusElect) error {
 // ProcessConsensusElectAccept processes an elect accept action.
 func (c *Channel) processConsensusElectAccept(data messagedata.ConsensusElectAccept) error {
 
+	err := data.Verify()
+	if err != nil {
+		return xerrors.Errorf("failed to process consensus#elect_accept message: %v", err)
```

Comment:
The error message seems off. We didn't failed to process, the message is invalid (and this is not our fault).
Same bellow.